profiles/common/modules/custom/nexteuropa_token/src/Entity/UrlTokenHandler.php

```diff
@@ -47,10 +47,11 @@ public function hookTokens($type, $tokens, array $data = array(), array $options
     $replacements = array();
 
     if ($this->isValidTokenType($type)) {
+      $token_types = token_get_entity_mapping('token');
```

Comment:
Function `token_get_entity_mapping()` by default provides mapping for the 'token' value;